bacom-blog/figure/images-to-figure.js

```diff
@@ -0,0 +1,17 @@
+import { selectAllBlocks } from '../../utils/mdast-utils.js';
+import { select } from 'unist-util-select'
+
+/**
+ * Convert all images to figures by change table name from "Images" to "Figure"
+ *
+ * @param {object} mdast - mdast tree
+ * @returns {object} mdast tree
+ */
+export async function imageToFigure(mdast) {
+  selectAllBlocks(mdast, 'Images').forEach((imageBlock) => {
+    select('text', imageBlock).value = 'Figure';
+  });
+
+  return mdast;
```

Comment:
small detail here, but this function made me realize that for the most part we are mutating the tree, not making copies of it. Did we need to return mdast if we are not saving it and passing it to another function? Maybe we return true?

Reply:
It might be useful to return some kind of report or true/false if we were able to migrate without issue.